Generate a variable dictionary and values

lfc-256 opened the pull request:
This dictionary include all parameters that we need to use in Matlab and Scilab.

Resolves-Issue #9

lfc-256 committed:
TASK: Create a variable dictionary and values

This dictionary include all parameters that we need to use in Matlab

Resolves: Issue-9

Signed-off-by: lfc-256 <luisf_cm@outlook.com>

jcontrolresearch reviewed (changes requested):
Please include a header to provide information about dictionary, for instance:

![image](https://user-images.githubusercontent.com/57429237/82840784-659a0900-9e99-11ea-907c-4d7092ab5a30.png)

split parameters on control parameters/ vehicle parameters/ trailer parameters as comments and variables in each section

lfc-256 committed:
TASK: Provide better organization of information

The changes include header of information about the functionality of
dictionary and a sections for contro parameters/vehicle
parameters/trailes parameters.

Resolves: better interpretation of information

Signed-off-by: lfc-256 <luisf_cm@outlook.com>

lfc-256 committed:
TASK: Remove script for Scilab

We only use Matlab for simulation

Resolves: Eliminate Scripts for Scilab

lfc-256 commented:
The changes to provide better organization of information and remove scripts for Scilab have been made.

lfc-256 committed:
TASK: Provide better organization of information

The changes include header of information about the functionality of
dictionary and a sections for control parameters/vehicle
parameters/trailers parameters.

Resolves: better interpretation of information

Signed-off-by: lfc-256 <luisf_cm@outlook.com>

lfc-256 commented:
Hi Julio, I made the changes  and this generate conflicts with the merge pull request. What can I do? Thanks!!!

jcontrolresearch reviewed (changes requested):
Use %% to header comment in order to define scrip information to future script external calls

jcontrolresearch commented:
> Hi Julio, I made the changes and this generate conflicts with the merge pull request. What can I do? Thanks!!!

@lfc-256 please click on highlighted section to see solution

![image](https://user-images.githubusercontent.com/57429237/82944953-baa15200-9f61-11ea-99e5-aba8e66a0ca5.png)

jcontrolresearch commented on Scripts/variable_dictionary.m:
```diff
@@ -1,5 +1,20 @@
+%{
```
Here must be %%

jcontrolresearch commented on Scripts/variable_dictionary.m:
```diff
@@ -1,5 +1,20 @@
+This scripts provides a variable dictionary with values of control
+parameters, vehicle parameters and trailer parameters that are used in the
+simulation .
+%}
```
Here to... %% please see [Code section spec Matlab](https://www.mathworks.com/help/matlab/matlab_prog/run-sections-of-programs.html)

jcontrolresearch reviewed (changes requested):
Team @HugoLeyvaOax , @SaraML , @raul-rd  please take a look in [code section for matlab scritps](https://www.mathworks.com/help/matlab/matlab_prog/run-sections-of-programs.html) for further develop scripts

HugoLeyvaOax commented:
Very Well

lfc-256 committed:
TASK: Provide better organization of information 2

The changes include the use if "%%" instead of "%{ %}" to header
comment.

Resolve: Conflicts files with branch

Signed-off-by: lfc-256 <luisf_cm@outlook.com>

lfc-256 committed:
Merge branch 'master' into Generate_a_variable_dictionary_and_values

Signed-off-by: lfc-256 <luisf_cm@outlook.com>

lfc-256 commented:
Thanks! I made the changes and follow the next steps [solution](https://github.com/githubteacher/github-for-developers-sept-2015/issues/648#issuecomment-144488223) and it worked.

jcontrolresearch commented on Scripts/variable_dictionary.m:
```diff
@@ -0,0 +1,16 @@
+%% This scripts provides a variable dictionary with values of control parameters, vehicle parameters and trailer parameters that are used in the simulation.
+
```
you forgot close the header with %% :)

jcontrolresearch reviewed (changes requested):
Follow this structure :)

[here](https://github.com/lfc-256/ControlBDA/pull/18#pullrequestreview-417862759)

lfc-256 committed:
TASK:Provide better organization of information 3

The changes include the correct use of "%%" to header comment.

Resolves: Correct use of the structure

Signed-off-by: lfc-256 <luisf_cm@outlook.com>

lfc-256 committed:
TASK: Add name file in header

the change include is the name file variable_dictionary.m
within the header structure

Resolves: What script is described

Signed-off-by: lfc-256 <luisf_cm@outlook.com>

jcontrolresearch reviewed (approved):
Great !
